installer/manifests/keptn/charts/control-plane/templates/mongodb-datastore.yaml

```diff
@@ -86,25 +86,10 @@ spec:
               optional: true
         - name: LOG_LEVEL
           value: {{ .Values.logLevel | default "info" }}
-        {{- include "control-plane.common.container-security-context" . | nindent 8 }}
-      - name: distributor
-        image: {{ .Values.distributor.image.repository }}:{{ .Values.distributor.image.tag | default .Chart.AppVersion }}
-        {{- include "control-plane.dist.livenessProbe" . | nindent 8 }}
-        {{- include "control-plane.dist.readinessProbe" . | nindent 8 }}
-        imagePullPolicy: IfNotPresent
-        ports:
-          - containerPort: 8080
-        {{- include "keptn.distributor.resources" . | nindent 8 }}
-        env:
-          - name: PUBSUB_IMPL
-            value: nats
-          - name: PUBSUB_TOPIC
-            value: 'sh.keptn.>'
-          - name: PUBSUB_RECIPIENT
-            value: '127.0.0.1'
-          - name: PUBSUB_RECIPIENT_PATH
-            value: '/event'
-          {{- include "control-plane.dist.common.env.vars" . | nindent 10 }}
+        - name: PUBSUB_IMPL
```

Comment:
Do we actually need this variable? It isn't used anywhere

Reply:
yeah this was used by distributor to decide between HTTP and nats but indeed not needed